config/config.yaml

```
@@ -209,6 +209,10 @@ branch-protection:
           branches:
             master:
               protect: true
+        plugin-sdk-go:
+          branches:
+            master:
```

```suggestion
            main:
```
I have noticed right now that the plugin-sdk-go default branch is `main` (and not master).

---

Ok fixed.